.github/label-actions.yml

```diff
@@ -0,0 +1,91 @@
+# Configuration for Label Actions - https://github.com/dessant/label-actions
+
+# When a discussion is resolved, lock the discussion
+'status/resolved':
+  discussions:
+    close: true
+    close-reason: 'resolved'
+    lock: true
+
+# Request more information from the author
+'auto/needs-more-info':
+  comment: >
+    :wave: @{issue-author}, please provide the requested information.
+  issues:
+    label: 
+      - 'status/needs-more-info' 
+  prs:
+    label: 
+      - 'status/needs-more-info' 
+
+# Thank the author for their response, remove auto-labels
+'auto/got-info':
+  comment: >
+    :wave: @{issue-author}, thank you for your response!
+  issues:
+    unlabel: 
+      - 'auto/needs-more-info'
+      - 'status/needs-more-info'
+  prs:
+    unlabel: 
+      - 'auto/needs-more-info'
+      - 'status/needs-more-info'
+
+# Comment that the issue will be transferred to another repository
+'auto/transfer':
+  comment: >
+    :wave: @{issue-author}, this issue will be transferred to another repository.
+  lock: true
+  lock-reason: 'off-topic'
+  issues:
+    label: 
+      - 'needs/transfer'
+      - 'status/wontfix'
+
+# Thank the author for their contribution
+'auto/thank-you':
+  comment: >
+    :wave: @{issue-author}, thank you for your contribution!
+
+# Mark issue for conversion to discussion
+'auto/make-discussion':
+  comment: >
+    :wave: @{issue-author}, this issue will be converted to a discussion. If you think this was a mistake, please let us know.
+  issues:
+    label: 
+      - 'needs/convert-to-discussion'
```

Comment:
I don't think this one is necessary, we already have a feature to convert an issue to discussion: https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#converting-an-issue-to-a-discussion

![image](https://github.com/docker/buildx/assets/1951866/6a679edf-cbe4-4483-b5ce-c1ece1d221ba)

Reply:
Agreed -- removing